addons/l10n_lv/Readme.rtf

```diff
@@ -0,0 +1 @@
+

```

Comment:
?

Reply:
updated in repository with description in english and latvian